### DISCUSSION TITLE
glide-data-grid

### DISCUSSION THREAD

**Discussion by xzrokeman** [zauberzeug/nicegui#3820] [Ideas / Feature Requests]
Streamlit has 2 great components: st.dataframe and st.data_editor, both using glide-data-grid under the hood. Glide-data-grid is better on accessiblity support than aggrid, which means a lot for average user for the UX is very familiar and consistent compare to Excel.
I'm not a javascript guy. I've tried to go through the ui.aggrid implementation. Hope someone could make a tutorial to help digging more details about wrapping 3rd party js libs using nicegui.

**Comment by falkoschindler** [comment]
Hi @xzrokeman,
[Glide Data Grid](https://grid.glideapps.com/) looks really nice! But integrating and supporting yet another element for tabular data would take much effort. Maybe it's a good project for a separate package in a separate repository like we did with [NiceGUI Highcharts](https://github.com/zauberzeug/nicegui-highcharts)?
